app/src/main/java/chat/rocket/android/authentication/login/presentation/LoginPresenter.kt

```diff
@@ -53,4 +56,15 @@ class LoginPresenter @Inject constructor(private val view: LoginView,
     fun signup() {
         navigator.toSignUp(navigator.currentServer!!)
     }
+
+    private suspend fun registerPushToken() {
+        // TODO: put it on constructor
```

Comment:
This code could be more generic? I'm sure we'll have to call in different activities later.

Reply:
I agree this has to be discussed better later since we're using coroutines here we don't have callbacks.